### PR TITLE
added initial support for numpy ndarrays and scalars

### PR DESCRIPTION
**Pull request by BorisTasevski**
### Description

added initial support for numpy ndarrays and scalars

### Pull request type

Please delete options that are not relevant.

- [X] Bugfix

### Related issues

Closes (link to issue)

[to_cypher_value badly converts numpy objects](https://github.com/memgraph/gqlalchemy/issues/195)

### CODE REVIEW

**Comment by katarinasupe** [comment]
Does this change need to be added [**here**](https://github.com/memgraph/gqlalchemy/blob/main/gqlalchemy/models.py#L145) too? So that it works when the property of a node is ndarray instead of a list in the class definition?

File: tests/test_utilities.py [hunk 1]
```diff
@@ -37,6 +38,9 @@
         ({"k1": 123, "k2": {"subkey1": "abc"}}, "{k1: 123, k2: {subkey1: 'abc'}}"),
         (None, "null"),
         (True, "True"),
+        (np.array([1, 2, 3]), "[1, 2, 3]"),
+        (np.array(["1", "2", "3"]), "['1', '2', '3']"),
+        (np.array([[1, 2], [3, 4], [4, 5]]), "[[1, 2], [3, 4], [4, 5]]"),
```
Comment (Josipmrden) [inline tests/test_utilities.py:43]: can you add one for floats?

**Review by Josipmrden — COMMENTED** [review]
minor comment

**Comment by BorisTasevski** [comment]
> Does this change need to be added [**here**](https://github.com/memgraph/gqlalchemy/blob/main/gqlalchemy/models.py#L145) too? So that it works when the property of a node is ndarray instead of a list in the class definition? Otherwise, all seems fine.

This closes the issue within the transformation module, we can create a separate task to look into the models

**Comment by katarinasupe** [comment]
> > Does this change need to be added [**here**](https://github.com/memgraph/gqlalchemy/blob/main/gqlalchemy/models.py#L145) too? So that it works when the property of a node is ndarray instead of a list in the class definition? Otherwise, all seems fine.
> 
> This closes the issue within the transformation module, we can create a separate task to look into the models

Okay, I opened a new issue for the models, so we don't forget.

**Review by Josipmrden — APPROVED** [review]

**Review by katarinasupe — APPROVED** [review]

**Pull request merged**